Assets/MixedRealityToolkit.Examples/Demos/Solvers/Scripts/SolverExampleManager.cs

```diff
@@ -96,6 +98,13 @@ private void RefreshSolverHandler()
             if (handler != null)
             {
                 handler.TrackedTargetType = TrackedType;
+
+#if WINDOWS_UWP || UNITY_EDITOR
+                // If using input simulation in editor or on Windows Mixed Reality platform, forward vector points through the fingers, not the palm.
+                // Apply additional rotation to correct this so that forward will be through the palm and outward
+                handler.AdditionalRotation = (TrackedType == TrackedObjectType.HandJoint) ? HandJointRotationFix : Vector3.zero;
```

Comment:
A bunch of random thoughts here:

I think I'm having a slight mental disconnect on this because these solver things are explicitly trying to track to a specific joint, and currently the code is doing exactly that - this fix is kinda changing the mechanics of what joint tracking is (i.e. instead of honoring the values from the platform, instead being like "actually, we want palm outward, let's try to massage the data to conform to that requirement). If we actually want "palm outward tracking" it feels like this should be a specific concept that we surface (i.e. maybe we have "meta joints" which share the the same location, but are maybe called "PalmMetaJointOutward" where we can then do some cross products and figure out the orientation ourselves).

Otherwise, we end up having to do per-platform fixes (and potentially re-fixes as the platform changes things or data gets "better").

That being said, I think this change is still better than no change (up going through the fingers doesn't make sense to anyone) so I will step off of my soapbox(?) now.
tl;dr if we

Reply:
All very good points. I was having a similar discussion in my head when looking at this issue/change. I originally looked at changing code in the solver(s) or SolverHandler. But I didn't want to have to deal with a per-platform edge case like this at the core.

Thus, I understand your concerns but I don't understand why you have them for this class, `SolverExampleManager`?

As a platform, we are providing joint tracking as is. It would be the app developer that is modifying the transform themselves by configuring the AdditionalRotation property.

Thus, the responsibility is on them to do so and handle it per-platform as desired

Reply:
Ah oh no, I misread the SolverExampleManager as just SolverManager or otherwise thought of it as more foundational than it is.